backend/src/core/models/job-queue.ts

```diff
@@ -34,4 +34,11 @@ export class JobQueue extends Model<JobQueue> {
     allowNull: false,
   })
   status!: JobStatus
+
+  @Column({
+    type: DataType.DATE,
+    allowNull: false,
+    defaultValue: new Date(),
+  })
+  visibleAt?: Date
```

Comment:
actually I'm a little confused, if `visibleAt` is not nullable and has a `defaultValue`, surely it should not be undefinable?

would be good to refactor the other columns in accordance to the PRs associated with [this issue](https://github.com/opengovsg/postmangovsg/issues/1696)

Reply:
```suggestion
  visibleAt: Date
```

Reply:
done